docs/2.0/docs/accountfactory/concepts/delegated-repositories.md

```diff
@@ -0,0 +1,38 @@
+# Delegated Repositoires
+
+:::note
+Delegated Repositories are only available to DevOps Foundations Enterprise customers.
```

Comment:
Our wording here is really tricky. 

You can see [here](https://github.com/gruntwork-io/docs/blob/main/docs/2.0/docs/pipelines/guides/setup-delegated-repo.mdx) that I tried to call this "Account Factory vended delegated repos" but maybe we need a new term for what that that other thing is. Users can have some amount of infrastructure delegation without being enterprise customers, they just don't get Account Factory vending them as part of account creation.

Whatever we call the other thing, it's an important thing for users to know about, as many want to know how to handle things like application deployments that require AWS access to push artifacts, etc, even if they aren't managing IaC.